src/esn.calendar.libs/app/components/event-date-edition/event-date-edition.pug

```diff
@@ -24,7 +24,16 @@
                 on-time-change="ctrl.onStartDateTimeChange",
                 locale="{{ ::ctrl.locale }}",
                 time-format="{{ ::ctrl.timeFormat }}",
-                disabled="ctrl.disabled"
+                disabled="ctrl.disabled",
+                ng-if="!ctrl.isMobile"
+              )
+              input.input.form-control(
+                type="time",
+                ng-model="ctrl.startTime",
+                ng-change="ctrl.onStartDateTimeChange()",
+                ng-blur="ctrl.dateOnBlurFn()",
+                ng-disabled="ctrl.full24HoursDay || ctrl.disabled",
```

Comment:
Why ng-disabled="ctrl.full24HoursDay" ?
Line should be hidden in this case

Reply:
fixed thanks.